integrations/logging/loki/install.sh

```diff
@@ -81,23 +95,52 @@ set -e
 # Deploy loki
 helm upgrade --install --create-namespace --namespace ${NAMESPACE} --values values.loki.yaml loki grafana/loki \
  --set "loki.storage.s3.secretAccessKey=${MINIO_PWD}" \
- --set "loki.basic_auth.password=${LOKI_PWD}"
+ --set "loki.basic_auth.password=${LOKI_PWD}" \
+ --set "loki.basic_auth.username=${LOKI_USERNAME}"
 
 # Deploy Grafana
 if [ "${GRAFANA}" == "True" ]; then
   helm upgrade --install --create-namespace --namespace ${NAMESPACE} --values values.grafana.yaml grafana grafana/grafana
 fi
 
-helm upgrade --install --wait --create-namespace \
-    --namespace ${NAMESPACE} \
-    logging-operator oci://ghcr.io/kube-logging/helm-charts/logging-operator
-kubectl rollout status deployment --namespace=${NAMESPACE} --timeout=180s
+if [ "${VECTOR}" == "True" ]; then
+
+  # Create kubearchive-vector namespace if it doesn't exist
+  kubectl create namespace kubearchive-vector --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
I think we should use the ${NAMESPACE} for vector deployment too

Reply:
Done